ts/features/bonus/siciliaVola/components/CheckIncomeComponent.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import { useRef } from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { SafeAreaView, ScrollView } from "react-native";
+import { View } from "native-base";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../components/core/typography/H1";
+import { GlobalState } from "../../../../store/reducers/types";
+import {
+  svGenerateVoucherBack,
+  svGenerateVoucherCancel,
+  svGenerateVoucherUnderThresholdIncome
+} from "../store/actions/voucherGeneration";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import { navigateToSvKoCheckIncomeThresholdScreen } from "../navigation/actions";
+import I18n from "../../../../i18n";
+import {
+  RadioButtonList,
+  RadioItem
+} from "../../../../components/core/selection/RadioButtonList";
+
+type OwnProps = {
+  onContinuePress: () => void;
+};
+type Props = OwnProps &
+  ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const getCheckIncomeUnderThresholItems = (): ReadonlyArray<
+  RadioItem<boolean>
+> => [
+  {
+    label: I18n.t("bonus.sv.voucherGeneration.checkIncome.threshold.under"),
+    id: true
+  },
+  {
+    label: I18n.t("bonus.sv.voucherGeneration.checkIncome.threshold.over"),
+    id: false
+  }
+];
+
+const CheckIncomeComponent = (props: Props): React.ReactElement => {
+  const [incomeUnderThreshold, setIncomeUnderThreshold] = React.useState<
+    boolean | undefined
+  >();
+
+  const elementRef = useRef(null);
```

Comment:
```suggestion
  const elementRef = useRef(null);
```
What does it needed for?

Reply:
Useless, removed in [e8d7037](https://github.com/pagopa/io-app/pull/3278/commits/e8d7037826a9a2762dc2d4e59e7ba51ab1fb9ad9)